### PR TITLE
adding possible observable embedding solution

### PR DESCRIPTION
**Pull request by stevenweaver**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,10 @@
   "name": "sars-cov-2-docs",
   "version": "1.0.0",
   "description": "",
-  "dependencies": {},
+  "dependencies": {
+    "@observablehq/runtime": "^4.7.0",
+    "@stevenweaver/case-vs-sequence-count": "https://api.observablehq.com/@stevenweaver/case-vs-sequence-count.tgz?v=3"
```
Comment (dannon) [inline package.json]: How does the versioning work here; that is, when you edit the root observable do we have to bump this `3` each time?

**Comment by dannon** [comment]
This is super cool.  Looks like there's a bit of bad merge content, and I think we could include some styling for the observables -- just let me know if I can help out.

**Comment by dannon** [comment]
@stevenweaver Thanks for the push here; this was super useful.  Check out #81's integation, which seems to be working really well in my hands.  I included the relevant commit from this there and added styling, etc.

**Comment by stevenweaver** [comment]
Great! The dynamic load is definitely the way to go. Sorry for the delayed response earlier, I somehow missed the earlier comments.

**Pull request closed without merging**